ar3:
    hardware_driver:
        serial_port: "COM4" # /dev/ttyACM0
        baudrate: 115200
        encoder_steps_per_deg: # encoder steps per joint angle degree
            joint_1: 227.5555555555556 # default 227.5555555555556
            joint_2: 284.4444444444444 # default 284.4444444444444 # joint enc 22.75555555555556
            joint_3: 284.4444444444444 # default 284.4444444444444
            joint_4: 223.0044444444444 # default 223.0044444444444
            joint_5: 56.04224675948152 # default 56.04224675948152 # joint enc 22.75555555555556
            joint_6: 108.0888888888889 # default 108.0888888888889
